community/modules/scheduler/schedmd-slurm-gcp-v6-login/main.tf

```diff
@@ -35,6 +35,7 @@ locals {
     }
   ]
 
+  public_access_config = [{ nat_ip = null, network_tier = "STANDARD" }]
```

Comment:
I think we should set network_tier=null. Default in terraform provider is PREMIUM. Public docs say "Use Premium Tier to optimize for performance". In vm-instance and previous slurm v5 login module we set network_tier to null which I assume is same as leaving to default of premium.